.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<!--
```

Comment:
Do we want to link to notion ticket, to have more context?

Reply:
i don't think so because the repo is public and notion links are broken to people without access to notion :/
happy to link them if our tickets are on github